jenkins-scripts/docker/lib/docker_generate_dockerfile.bash

```diff
@@ -348,6 +348,10 @@ cat >> Dockerfile << DELIM_NVIDIA2_GPU
     ${NVIDIA_VISIBLE_DEVICES:-all}
   ENV NVIDIA_DRIVER_CAPABILITIES \
     ${NVIDIA_DRIVER_CAPABILITIES:+$NVIDIA_DRIVER_CAPABILITIES,}graphics
+DELIM_NVIDIA2_GPU
+
+if [[ ${LINUX_DISTRO} == 'ubuntu' ]] && [[ ${DISTRO} == 'bionic' && ${DISTRO} == 'focal' ]]; then
```

Comment:
this logic looks like `(LINUX_DISTRO is ubuntu and (DISTRO is bionic and DISTRO is focal))`

I think one of the `and` operators should be an `or`: `DISTRO is bionic OR DISTRO is focal`

Reply:
ouch, good catch. [68ea0c0](https://github.com/ignition-tooling/release-tools/pull/715/commits/68ea0c03b60336db8827aa0c8667de0b2a32caf6)